tests/components/panasonic_viera/test_init.py

```diff
@@ -56,6 +56,21 @@ async def async_get_device_info():
 
     mock_remote.async_get_device_info = async_get_device_info
 
+    async def async_turn_on():
+        return
+
+    mock_remote.async_turn_on = async_turn_on
```

Comment:
Set these as async mocks so we can assert the calls to the remote mock.
```suggestion
    mock_remote.async_turn_on = AsyncMock(side_effect=async_turn_on)
```

Reply:
great! thx for helping on this one, will try to do it later

Reply:
thanks for the tip on this one!! will make a commit soon